hw_7/functional.py

```diff
@@ -0,0 +1,79 @@
+import sys                                   # for 7 task
+
+
+# I task - sequential map
+def sequential_map(*args):
+    if len(args[-1]) == 0:                   # check of list attendance
+        return None
+    temp = map(args[0], args[-1])            # execution of first function
+    for function in args[1:-1]:
+        temp = map(function, temp)           # execution of every function
+    return list(temp)
```

Comment:
В целом суть верная, я бы вот так написал
```suggestion
def sequential_map(*args):
    *functions, container = args
    for function in functions:
        container = map(function, container)           # execution of every function
    return list(container)
```